modules/cli/pages/mctimings.adoc

```diff
@@ -93,501 +73,331 @@ If using a secure connection (the `-s` flag) then this should be 11211.
 | The password of the administrator who is authenticating.
 
 | `-s`
-| Use a secure connection to retrieve timing statistics.
+| Specifies that a secure connection is used.
 
 | `-v`
-| Show operation timings as histograms.
-If this flag is used, one or more operation names or associated hexadecimals can be specified; so as to limit output to that which corresponds to those operations.
-Hexadecimals must be specified within _back-ticks_.
-If this flag is not used, then only the total number of operations are displayed.
+| Specifies that operation timings are displayed as histograms.
+
+Whenever `-v` is used, one or more operation names can also be specified; so as to restrict output to that corresponding to the specified operations.
+
+If `-v` is _not_ used, the displayed output consists only of a _total_ figure for each of all operations performed.
```

Comment:
This flag is also overloaded for when the --json argument is used. -v will pretty print the json output.